Proofreading MO Guide (#11605)
openvinotoolkit/openvino: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Bert_ner.md

# Converting PyTorch BERT-NER Model {#openvino_docs_MO_DG_prepare_model_convert_model_pytorch_specific_Convert_Bert_ner}

> ## Downloading and Converting the Model to ONNX

Reviewer: ```suggestion
Goal of this article is to present you with a step-by-step guide on how to convert PyTorch BERT-NER Model to IR. First of all, it is required to download and convert the model to ONNX.

## Downloading and Converting the Model to ONNX
```

# Converting a PyTorch BERT-NER Model {#openvino_docs_MO_DG_prepare_model_convert_model_pytorch_specific_Convert_Bert_ner}

> Goal of this article is to present you with a step-by-step guide on how to convert PyTorch BERT-NER Model to IR. First of all, it is required to download and convert the model to ONNX.

Reviewer 2: ```suggestion
The goal of this article is to present a step-by-step guide on how to convert PyTorch BERT-NER model to OpenVINO IR. First, you need to download the model and convert it to ONNX.

```
